src/Mod/OpenSCAD/CMakeLists.txt

```diff
@@ -59,3 +101,7 @@ INSTALL(
        "${CMAKE_INSTALL_DATADIR}/Mod/OpenSCAD/Resources/icons"
 )
 
+INSTALL(FILES ${OpenSCADTests_SRCS} DESTINATION Mod/OpenSCAD/OpenSCADtest)
```

Comment:
OpenSCADtest must become OpenSCADTest as otherwise a "make installed" FreeCAD will fail to load the test modules.

Reply:
I completely forgot to test the install target, sorry about that! Fixed now.